cmd/containerd-shim-runc-v2/process/io.go

```diff
@@ -27,16 +27,19 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
-	"sync"
-	"sync/atomic"
-	"syscall"
-	"time"
 
+	"github.com/containerd/containerd/v2/defaults"
+	"github.com/containerd/containerd/v2/pkg/cio"
 	"github.com/containerd/containerd/v2/pkg/namespaces"
 	"github.com/containerd/containerd/v2/pkg/stdio"
 	"github.com/containerd/fifo"
 	runc "github.com/containerd/go-runc"
 	"github.com/containerd/log"
+
+	"sync"
+	"sync/atomic"
+	"syscall"
+	"time"
```

Comment:
Probably need to run`gofmt`. these should be together with other std imports above.

Reply:
CI linters are green 🤔

Reply:
The std imports should be in first group with `path/filepath`.

Reply:
Fixed